File: Data Structures & Algorithms/two-integer-sum/submission-3.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] output = new int[2] ;
        for(int i=0; i<nums.length-1; i++){
            int j = i+1;
            while(j<nums.length){
                if(nums[i]+nums[j]==target){
                    output[0]=i;
                    output[1]=j;
                }
                j++;
            }
            // for(int j=i+1; j<nums.length;j++){
            //     if(nums[i]+nums[j]==target){
            //         output[0]=i;
            //         output[1]=j;
            //     }
            // }
        }
    return output;
    }
}

File: Data Structures & Algorithms/anagram-groups/submission-3.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        strs_dict =  {}
        output = []
        for s in strs:
            sorted_string = "".join(sorted(s))
            # print(sorted_string)
            if sorted_string in strs_dict:
                strs_dict[sorted_string].append(s)
            else:
                strs_dict[sorted_string] = [s]
        # print(strs_dict)
        for value in strs_dict.values():
            output.append(value)
        return output
        